### DISCUSSION TITLE
Unable to verify academic affiliation: ASU Online

### DISCUSSION THREAD

**Discussion by DesVoeux** [community/community#50264] [GitHub Education]
I've submitted all possible documents to verify my enrollment at Arizona State University but I keep getting the following rejection response:

"You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so."

I'll note that both the dated tuition receipt and the screenshot of my school portal clearly show I attend classes online. It feels like a situation that a human being would instantly understand but the software can't handle.

**Comment by jsuddsjr** [comment]
Maybe it needs a few more examples for its training algorithm. Not surprisingly, online learning has only increased since 2020, so I'm surprised there's even a requirement for signing up in a geographic location. That's so 2019.
